data-model/fdo-profiles/0.1.0/tombstone/schema/tombstone-request.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$id": "https://json-schema.org/draft/2020-12/schemas/fdo-profiles/0.1.0/tombstone-request.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$comment": "Schema for tombstoning requests to DiSSCo PID API",
+  "tombstoneText": {
+    "type": "string",
+    "description": "Free text describing why the object was archived",
```

Comment:
_"Free text describing why the object was archived"_. Shouldn't it be "Free text describing why the object was tombstoned"?